t/13-Config/30-min_app_githooks_version.t

```diff
@@ -0,0 +1,55 @@
+#!perl
+
+use strict;
+use warnings;
+
+use App::GitHooks;
+use App::GitHooks::Test;
+use App::GitHooks::Utils;
+use Test::Exception;
+use Test::FailWarnings -allow_deps => 1;
+use Test::Requires::Git;
+use Test::More;
+
+plan(tests => 6);
+
+# Require git.
+test_requires_git( '1.7.4.1' );
+
+App::GitHooks::Test::ok_reset_githooksrc(
+    content => ''
+);
+
+my $app = App::GitHooks->new(
+    arguments => [],
+    name      => 'commit-msg',
+);
+
+ok($app->get_config(),'config is retrieved when no min_app_githooks_version is specified');
```

Comment:
Since we're testing that an exception isn't raised incorrectly, would it make sense to specifically test for that? e.g.:

```
lives_ok(
    sub {
        my $app = App::GitHooks->new(
            arguments => [],
            name      => 'commit-msg',
        );
    },
    'Config is retrieved when no min_app_githooks_version is specified',
);
```